net/graphsync_fetcher_test.go

```diff
@@ -571,6 +578,21 @@ type fakePeerTracker struct {
 	peers []*types.ChainInfo
 }
 
+func newFakePeerTracker(cis ...*types.ChainInfo) *fakePeerTracker {
+	return &fakePeerTracker{
+		peers: cis,
+	}
+}
+
 func (fpt *fakePeerTracker) List() []*types.ChainInfo {
 	return fpt.peers
 }
+
+func (fpt *fakePeerTracker) Self() peer.ID {
+	return peer.ID("")
+}
+
+func requireBlockStorePut(t *testing.T, bs bstore.Blockstore, data format.Node) {
```

Comment:
Copy from `net/fetcher_test.go`? Not seeing where this is used.

Reply:
yes, used in `graphsync_fetcher_test.go`